### DISCUSSION TITLE
How to configure sensibility on Dock?

### DISCUSSION THREAD

**Discussion by gazambuja** [elementary/triage#579] [Q&A]
When hide (window cover), the Dock shows very fast, and do not let me click on some link or button on the end of the window (like the browser and click on some link at the end).
How can I make it less sensitive to the mouse position?

**Comment by jeremypw** [comment]
Try switching "Pressure Reveal" on in Settings/Desktop/Dock & Panel.  This only works on some devices.

**Reply by gazambuja** [reply]
I do not have that option. (this is not a laptop)

**Reply by jeremypw** [reply]
Do you mean the switch does not appear in Settings/Desktop & Panel or that it makes no difference?  Which version of Elementary are you using?

**Comment by gazambuja** [comment]
The option do not appear:

![Screenshot from 2024-10-02 15 02 29](https://github.com/user-attachments/assets/cbb25dd2-50f3-41af-b93a-e015d487ac1d)

Elementary 8, updated.

**Reply by jeremypw** [reply]
Hmm, it seems to be removed in OS8.  Its a rewritten dock and the window manager has changed a lot too. But this is definitely an issue that needs raising - the dock is too sensitive and appears before the pointer reaches the edge of the window which is annoying. One work around at the moment is to choose "Never hide" for the dock, which stops windows overlapping it - but that wastes space.

**Comment by jeremypw** [comment]
Not sure whether this needs fixing in Dock or in Gala (window manager) so probably raise in `elementary/triage` first?
